ieeeuoft/hackathon-template
# Remove inventory filter from drawer on desktop

## Overview

- On the inventory page on desktop, the filter will no longer be on a drawer to match the redesign on Figma
- Clicking on an item will open up the Product Overview drawer (with a static layout/item information)
- An CloseIcon will appear on the search bar, once we hook up Formik to it, we should conditionally render the icon to appear only if the input has a value


## Unit Tests Created

N/A


## Steps to QA

- Go to the inventory page
- Confirm that the inventory filter looks the same as the [Figma](https://www.figma.com/file/1bYQy6ioaZwxPD9w6D47jW/Hardware-signout-site?node-id=2640%3A20442) mobile and desktop design
- Click on an item, the Product Overview drawer should popup
- An `x` icon should be on the search bar, but pressing it won't do anything
